requirements-dev.txt

```diff
@@ -4,3 +4,4 @@ pytest-cov
 coverage
 sphinx==2.2.1
 sphinx_rtd_theme==0.4.3
+paramiko
```

Comment:
This isn't required if the class always exists but may raise an exception if paramiko is missing (docs build and tests run without it)